src/components/FileHistory/HistoryModal.jsx

```diff
@@ -0,0 +1,102 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+
+import ExperimentalModal from 'cozy-ui/transpiled/react/Labs/ExperimentalModal'
+import Spinner from 'cozy-ui/transpiled/react/Spinner'
+import { translate } from 'cozy-ui/react/I18n'
+import { Caption } from 'cozy-ui/transpiled/react/Text'
+
+import { withClient } from 'cozy-client'
+import { withRouter } from 'react-router'
+
+import HistoryRow from './HistoryRow'
+import styles from './styles.styl'
+
+import { CozyFile } from 'models'
+import { isMobile } from 'cozy-device-helper/dist/platform'
+import { exportFilesNative } from 'drive/web/modules/navigation/duck/actions'
+const formatDate = (date, f) => {
+  return f(date, 'DD MMMM - HH[h]mm')
```

Comment:
Nit, but a `h` in there seems… very french :D

```suggestion
  return f(date, 'DD MMMM - HH:mm')
```